### PR TITLE
Add exit signing utility

### PR DESCRIPTION
**Pull request by eezcjkr**

### CODE REVIEW

File: lib/exit.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    vBuff.writeInt8(sig.v, 0);
+    return Buffer.concat([sigHashBuff, sig.r, sig.s, vBuff]);
+  }
+
+  // params:
+  //  utxoid: string with 0x, e.g. 0x0000000000000000000000000000000000b447b980aea5accb5fd68789f6b099
+  //  amount: plain JS number
+  // returns:
+  //  64 byte buffer
+  static sigHashBuff(utxoid, amount) {
```
Comment (johannbarbie) [inline lib/exit.js]: doesn't this need to contain the buyer and the rate it is bought at?

Comment (eezcjkr) [reply lib/exit.js]: Amount = the value it is being sold at
The buyer could be anyone. The way I imagine it is I sign off my exit at some value and whoever submits this to the smart contract first will pay the amount and receive the exit.
So I could send it to you privately or just post it in some public place.

Comment (eezcjkr) [reply lib/exit.js]: I.E. I dont really care who the buyer is

Comment (johannbarbie) [reply lib/exit.js]: cool, got it. perfect

**Pull request merged**